components/raftstore/src/store/worker/disk_check.rs

```diff
@@ -72,7 +72,8 @@ impl Runner {
     /// Only for test.
     /// Generate a dummy Runner.
     pub fn dummy() -> Self {
-        Self::build(PathBuf::from("./").join(Self::DISK_IO_LATENCY_INSPECT_FILENAME))
+        let tmp = std::env::temp_dir();
+        Self::build(tmp.join(Self::DISK_IO_LATENCY_INSPECT_FILENAME))
```

Comment:
I think use tmp is not a good idea here because it can cause race when multiple use share the same instance. I suggest impl `Drop` for the Runner to remove this obsolescent file when the test is over.

Reply:
Let's drop the file in another PR.